lib/public/HtmlUrlChecker.js

```diff
@@ -31,6 +31,7 @@ function HtmlUrlChecker(options, handlers)
 			thisObj.currentCustomData = input.data.customData;
 			thisObj.currentDone = done;
 			thisObj.currentHtmlUrl = input.url;
+			resetHtmlLinkChecker(thisObj);
```

Comment:
I really don't like re-instantiating `HtmlChecker` on each link result. It should use a single instance.